ci-tests/terraform/lineprinter/lineprinter.go

```diff
@@ -24,12 +24,12 @@ type Print func(args ...interface{})
 // One use-case is connecting a subprocess's standard streams to a
 // logger:
 //
-//   linePrinter := &linePrinter{
-//     Print: &Trimmer{WrappedPrint: logrus.StandardLogger().Debug}.Print,
-//   }
-//   defer linePrinter.Close()
-//   cmd := exec.Command(...)
-//   cmd.Stdout = linePrinter
+//	linePrinter := &linePrinter{
```

Comment:
If those lines are in a comment - maybe they can be removed? 
also for lines 13-14

Reply:
This is an example of usage as part of the function docs